src/frequenz/sdk/config/_managing_actor.py

```diff
@@ -185,8 +187,8 @@ async def _run(self) -> None:
             async for event in file_watcher:
                 if not event.path.exists():
                     _logger.error(
-                        "%s: Received event %s, but the watched path %s doesn't exist.",
-                        self,
+                        "[%s] Received event %s, but the watched path %s doesn't exist.",
+                        self.name,
```

Comment:
is  `self.name` really needed in these logs? 
We don't use it like this anywhere and Logger includes name of the file in logs.

Reply:
Yeah, I guess you are right. The idea was that there could be many config managing actors, and without the name we don't know which one is actually emitting the logs, but since this will be probably always a singleton in one app, maybe is not really useful. Sometimes it is a delicate balance between the use we give this now and the use other users might give it in the future. If someone, for some reason, needs to instantiate more than one config managing actor, having the name is definitely useful.

Reply:
I'm undecided about this, so unless there is a strong push-back I prefer to leave it as it is now, we can change it in a future PR.